src/study.js

```diff
@@ -1,12 +1,46 @@
+import Glean from "@mozilla/glean/webext";
+import PingEncryptionPlugin from "@mozilla/glean/webext/plugins/encryption";
+
 import { Rally, runStates } from "@mozilla/rally";
 import { onPageData, stopMeasurement } from "./attention-reporter";
-  
+
+import * as pageMetrics from "../src/generated/page.js";
+import * as pageVisitMetrics from "../src/generated/pageVisit.js";
+import * as eventMetrics from "../src/generated/event.js";
+import * as pageAudioMetrics from "../src/generated/pageAudio.js";
```

Comment:
Given that we need `pageAttentionMetrics` to capture the scroll events and not `pageAudioMetrics`, this probably means you have to do (I'm guessing)
```suggestion
import * as pageAttentionMetrics from "../src/generated/pageAttention.js";
```

And I am also presuming that you'd have to probably regenerate the API as well, correct? I made a suggestion in `metrics.yaml` that probably requires regenerating.

Reply:
Yes, you're right!